cirq-core/cirq/experiments/t1_decay_experiment_test.py

```diff
@@ -165,6 +167,72 @@ def test_all_off_results():
     )
 
 
+def test_constant():
```

Comment:
It might be worth looking into parameterizing this test with `@pytest.mark.parameterize` so that you can remove some of the code duplication between the 100 and 400 cases, and just turn the relevant values between these two results into parameters for the test. Here's a quick example: https://github.com/quantumlib/Cirq/blob/master/cirq-core/cirq/ops/common_gates_test.py#L60

Reply:
Thanks for this suggestion! I've added this to the test, along with the explicitly exponential data from your other comment.